sycl/doc/extensions/IntelFPGA/FPGALsu.md

```diff
@@ -83,37 +160,45 @@ public:
 ```c++
 #include <sycl/ext/intel/fpga_extensions.hpp>
 ...
-cl::sycl::buffer<int, 1> output_buffer(output_data, 1);
-cl::sycl::buffer<int, 1> input_buffer(input_data, 1);
+sycl::buffer<int, 1> output_buffer(output_data, 1);
+sycl::buffer<int, 1> input_buffer(input_data, 1);
 
-Queue.submit([&](cl::sycl::handler &cgh) {
-  auto output_accessor = output_buffer.get_access<cl::sycl::access::mode::write>(cgh);
-  auto input_accessor = input_buffer.get_access<cl::sycl::access::mode::read>(cgh);
+Queue.submit([&](sycl::handler &cgh) {
+  auto output_accessor = output_buffer.get_access<sycl::access::mode::write>(cgh);
+  auto input_accessor = input_buffer.get_access<sycl::access::mode::read>(cgh);
 
   cgh.single_task<class kernel>([=] {
     auto input_ptr = input_accessor.get_pointer();
     auto output_ptr = output_accessor.get_pointer();
 
     using PrefetchingLSU =
-        cl::sycl::ext::intel::lsu<cl::sycl::ext::intel::prefetch<true>,
-                             cl::sycl::ext::intel::statically_coalesce<false>>;
+        sycl::ext::intel::lsu<sycl::ext::intel::prefetch<true>,
+                             sycl::ext::intel::statically_coalesce<false>>;
 
     using BurstCoalescedLSU =
-        cl::sycl::ext::intel::lsu<cl::sycl::ext::intel::burst_coalesce<false>,
-                             cl::sycl::ext::intel::statically_coalesce<false>>;
+        sycl::ext::intel::lsu<sycl::ext::intel::burst_coalesce<false>,
+                             sycl::ext::intel::statically_coalesce<false>>;
 
     using CachingLSU =
-        cl::sycl::ext::intel::lsu<cl::sycl::ext::intel::burst_coalesce<true>,
-                             cl::sycl::ext::intel::cache<1024>,
-                             cl::sycl::ext::intel::statically_coalesce<true>>;
+        sycl::ext::intel::lsu<sycl::ext::intel::burst_coalesce<true>,
+                             sycl::ext::intel::cache<1024>,
+                             sycl::ext::intel::statically_coalesce<true>>;
 
-    using PipelinedLSU = cl::sycl::ext::intel::lsu<>;
+    using PipelinedLSU = sycl::ext::intel::lsu<>;
 
     int X = PrefetchingLSU::load(input_ptr); // int X = input_ptr[0]
     int Y = CachingLSU::load(input_ptr + 1); // int Y = input_ptr[1]
 
     BurstCoalescedLSU::store(output_ptr, X); // output_ptr[0] = X
     PipelinedLSU::store(output_ptr + 1, Y);  // output_ptr[1] = Y
+
+    // latency controls
+    // Load is anchor 1
+    int Z = PrefetchingLSU::load(input_ptr + 2,
+                                 property_list{latency_anchor_id<1>});
+    // Store occurs 5 cycles after the anchor 1 read
+    BurstCoalescedLSU::store(output_ptr + 2, Z,
+                             property_list{latency_constraint<1, latency::exact, 5>});
```

Comment:
As for LSU types, we recommend using alias for them because their type names are heavily reused when calling their member functions, so alias would help eliminating duplications. 
However, as for their member functions, since latency control properties could be different in each call, alias would not help a lot here.